interfaces/uss_qualifier/uss_qualifier.yaml

```diff
@@ -0,0 +1,113 @@
+openapi: 3.0.2
+info:
+  title: Automated testing tool for USS Qualifier
+  version: 0.0.1
+  description: >-
+    This is an interface to run the qualifier tests.
+
+components:
+  schemas:
+    RunTests:
+      type: object
+      required:
+      - flight_records
+      - auth_spec
+      - user_config
+      properties:
+        flight_records:
+          description: >-
+            Comma separated list of flight record file names. Files should be uploaded by upload method before running the tests.
```

Comment:
We'll also need to define the endpoint(s) where these files can be uploaded -- this can be in a separate PR.